integration/test_ml.py

```diff
@@ -0,0 +1,380 @@
+# Copyright 2020 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Integration tests for firebase_admin.ml module."""
+import re
+import os
+import shutil
+import unittest
+import pytest
+
+
+from firebase_admin import ml
+from firebase_admin import exceptions
+from tests import testutils
+
+
+# pylint: disable=import-error,no-name-in-module
+try:
+    import tensorflow as tf
+    _TF_ENABLED = True
+except ImportError:
+    _TF_ENABLED = False
+
+
+NAME_ONLY_ARGS = {
+    'display_name': 'TestModel123'
```

Comment:
Again I highly recommend randomly generated names. We've been burnt by this sort of things before. Problem is we cannot always guarantee proper clean up of resources after tests. If the developer does a hard exit during tests, things can be left lying around, and the subsequent attempts to run the tests will fail. But this is also such a rare case, we don't want to check and clean up before each test run as that makes the test suite unnecessarily slow all the time.